mappings/net/minecraft/client/network/BlockListManager.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_6370 net/minecraft/client/network/BlockListManager
```

Comment:
This doesn't look like a block list *manager*. This instead is like resolving an address to remove all redirects and check if the un-redirected address is blocked.